includes/admin/class-wc-admin-setup-wizard.php

```diff
@@ -536,7 +536,7 @@ class="location-input wc-enhanced-select dropdown"
 	public function wc_setup_store_setup_save() {
 		check_admin_referer( 'wc-setup' );
 
-		// phpcs:disable WordPress.VIP.SuperGlobalInputUsage.AccessDetected, WordPress.VIP.ValidatedSanitizedInput.InputNotValidated, WordPress.VIP.ValidatedSanitizedInput.MissingUnslash
+		// phpcs:disable WordPress.VIP.SuperGlobalInputUsage.AccessDetected, WordPress.VIP.ValidatedSanitizedInput.InputNotValidated, WordPress.VIP.ValidatedSanitizedInput.MissingUnslash, WordPress.Security.ValidatedSanitizedInput
```

Comment:
I've just noticed all those disables... Maybe this block should be checked again. For example, unslashing is needed!

e.g.:

```
$address        = sanitize_text_field( $_POST['store_address'] );
```

to:

```
$address        = wc_clean( wp_unslash( $_POST['store_address'] ) );
```

Would you mind checking this? Also bare in mind the .VIP. rules no longer exist. I think it's .SECURITY. so they can be removed.